ras_party/views/account_view.py

```diff
@@ -73,3 +75,14 @@ def put_email_verification(token):
 def resend_verification_email(party_uuid):
     response = ras_party.controllers.account_controller.resend_verification_email(party_uuid)
     return make_response(jsonify(response), 200)
+
+
+@account_view.route('/respondents/add_survey', methods=['POST'])
+def respondent_add_survey():
+    payload = request.get_json() or {}
+    v = Validator(Exists('party_id', 'enrolment_code'))
+    if not v.validate(payload):
+        raise RasError(v.errors, 400)
+
+    response = ras_party.controllers.account_controller.add_new_survey_for_respondent(payload)
```

Comment:
add_new_survey_for_respondent is not returning anything so no need for assignment here